pep-0001.txt

```diff
@@ -204,10 +204,9 @@ The standard PEP workflow is:
     editors do not consider whether they seem likely to be accepted.
   * The title accurately describes the content.
   * The PEP's language (spelling, grammar, sentence structure, etc.)
-    and code style (examples should match :pep:`8` & :pep:`7`) should be
-    correct and conformant.  The PEP will be checked for formatting
-    (plain text or reStructuredText) by Travis CI, and will not be
-    approved until this passes.
+    and code style (examples should match :pep:`7` & :pep:`8`) should be
+    correct and conformant.  The PEP will be checked for reStructuredText
+    formatting by the repo's CIs, and will not be approved until this passes.
```

Comment:
Whilst CI is a term of art, PEP authors who are primarily scientists, typing specialists, etc may not be as familiar with it -- this wording attempts to focus on the what, rather than the how.

A

```suggestion
    correct and conformant.  The PEP text will be automatically checked for 
    correct reStructuredText formatting when the pull request is submitted. 
    PEPs with invalid reST markup will not be approved.
```